tests/functional/schema_tests/fixtures.py

```diff
@@ -0,0 +1,1397 @@
+{% macro current_timestamp() -%}
+  {{ return(adapter.dispatch('current_timestamp', 'test_utils')()) }}
+{%- endmacro %}
+
+{% macro default__current_timestamp() -%}
+  now()
+{%- endmacro %}
+
+"""
+
+test_utils__macros__custom_test_sql = """
+{% macro test_dispatch(model) -%}
+  {{ return(adapter.dispatch('test_dispatch', macro_namespace = 'test_utils')()) }}
+{%- endmacro %}
+
+{% macro default__test_dispatch(model) %}
+    select {{ adapter.dispatch('current_timestamp', macro_namespace = 'test_utils')() }}
+{% endmacro %}
+
+"""
+
+local_dependency__dbt_project_yml = """
+name: 'local_dep'
+version: '1.0'
+config-version: 2
+
+profile: 'default'
+
+macro-paths: ["macros"]
+
+"""
+
+local_dependency__macros__equality_sql = """
+{#-- taken from dbt-utils --#}
+{% test equality(model, compare_model, compare_columns=None) %}
+  {{ return(adapter.dispatch('test_equality')(model, compare_model, compare_columns)) }}
+{% endtest %}
+
+{% macro default__test_equality(model, compare_model, compare_columns=None) %}
+
+{% set set_diff %}
+    count(*) + abs(
+        sum(case when which_diff = 'a_minus_b' then 1 else 0 end) -
+        sum(case when which_diff = 'b_minus_a' then 1 else 0 end)
+    )
+{% endset %}
+
+{#-- Needs to be set at parse time, before we return '' below --#}
+{{ config(fail_calc = set_diff) }}
+
+{#-- Prevent querying of db in parsing mode. This works because this macro does not create any new refs. #}
+{%- if not execute -%}
+    {{ return('') }}
+{% endif %}
+-- setup
+{%- do dbt_utils._is_relation(model, 'test_equality') -%}
+{#-
+If the compare_cols arg is provided, we can run this test without querying the
+information schema — this allows the model to be an ephemeral model
+-#}
+
+{%- if not compare_columns -%}
+    {%- do dbt_utils._is_ephemeral(model, 'test_equality') -%}
+    {%- set compare_columns = adapter.get_columns_in_relation(model) | map(attribute='quoted') -%}
+{%- endif -%}
+
+{% set compare_cols_csv = compare_columns | join(', ') %}
+
+with a as (
+    select * from {{ model }}
+),
+b as (
+    select * from {{ compare_model }}
+),
+a_minus_b as (
+    select {{compare_cols_csv}} from a
+    {{ dbt_utils.except() }}
+    select {{compare_cols_csv}} from b
+),
+b_minus_a as (
+    select {{compare_cols_csv}} from b
+    {{ dbt_utils.except() }}
+    select {{compare_cols_csv}} from a
+),
+
+unioned as (
+
+    select 'a_minus_b' as which_diff, * from a_minus_b
+    union all
+    select 'b_minus_a' as which_diff, * from b_minus_a
+
+)
+
+select * from unioned
+
+{% endmacro %}
+
+"""
+
+case_sensitive_models__schema_yml = """
+version: 2
+
+models:
+  - name: lowercase
+    columns:
+      - name: id
+        quote: true
+        tests:
+          - unique
+  - name: uppercase
+    columns:
+      - name: id
+        quote: true
+        tests:
+          - unique
+
+"""
+
+case_sensitive_models__uppercase_SQL = """
+select 1 as id
+
+"""
+
+case_sensitive_models__schema_yaml = """
+
+"""
```

Comment:
Is this test testing for a yml and yaml file or is this just trash from the original test we can delete?  Based on the name it seems like trash.

Reply:
Did a search and looks like there's nothing using it so I removed it as suggested